docs/more/wasm-build-tools.md

```diff
@@ -0,0 +1,39 @@
+## **Wasm Build Tools**
```

Comment:
This file should have a title, description and be added to [sidebars.json](https://github.com/yewstack/yew/blob/master/website/sidebars.json) like the other ones

Reply:
Complete